WordPress/src/main/res/layout/filtered_list_component.xml

```diff
@@ -30,6 +30,21 @@
 
         </androidx.appcompat.widget.Toolbar>
 
+        <com.google.android.material.tabs.TabLayout
+            android:id="@+id/tab_layout"
+            android:background="@color/primary"
+            android:layout_gravity="start"
+            android:layout_height="@dimen/tab_height"
+            android:layout_width="match_parent"
+            app:tabGravity="fill"
+            app:tabIndicatorColor="@android:color/white"
```

Comment:
I'm not sure if we shouldn't use our color, what do you think?

Reply:
As per slack discussion let keep it with android color for alignment with tab layout in other parts of the app (for example the Notifications Fragment tabs).